README.md

```diff
@@ -22,11 +22,9 @@ Create a campus guide application to help students locate their classrooms.
 
 ## Installation Instructions
 1. Download and install [Android Studio](https://developer.android.com/studio)
-2. [Get an API key for Google Maps](https://developers.google.com/maps/documentation/android-sdk/get-api-key)
-3. Clone the repository onto your computer
-4. Create a copy of the file `app\src\debug\res\values\google_maps_api.example.xml` in the same folder.
-5. Rename that file `google_maps_api.xml`
-6. The file contains a single `<string>` tag. Change the `name` property from `google_maps_key_ex` to `google_maps_key`. Change the contents of the tag from `google_maps_key` to the value of your API key (it starts with `AIza`)
-7. Perform steps 4 through 6 with the file `app\src\release\res\values\google_maps_api.example.xml`. This step is only necessary to build the app in Release configuration.
-8. Open the project in Android Studio
-9. Run the project using an emulator or using a connected android device.
+1. [Get an API key for Google Maps](https://developers.google.com/maps/documentation/android-sdk/get-api-key)
+1. Clone the repository onto your computer
+1. Create a file `app\src\main\res\values\google_maps_api.xml`.
+1. Write the following to the file: `<resources><string name="google_maps_key">*API_KEY_GOES_HERE*</string></resources>`
```

Comment:
Originally we had two extra parameters in the string tag like so:

`<resources><string name="google_maps_key" translatable="false" templateMergeStrategy="preserve">*API_KEY_GOES_HERE*</string></resources>`

I'm pretty sure they should be kept

Reply:
I'll add those back. My B.